docs/pages/enroll-resources/mcp-access/stdio.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: MCP Access with Stdio MCP Server
+description: How to access MCP server with stdio transport.
```

Comment:
Should we have this page be specific to "stdio"? I would consider reframing it as general guide for registering your MCP server with Teleport, and adding an admonition at the beginning that only stdio transport is currently supported and that HTTP is coming soon. Cause once we add SSE/Streamable-HTTP, the process will be exactly the same, just the way you register the app is a little different?

Reply:
it's hard to imagine all differences those guides might have at the moment. at least the architecture will be very different for streamable-HTTP. SSE and streamable-HTTP will likely have some sections on "auth", and they may have more specific troubleshooting and links to auto-discovery.

to me it's leaner to maintain separate guides but i don't have a strong opinion.